pkg/types/ccipocr3/plugin_commit_types.go

```diff
@@ -15,20 +15,13 @@ package ccipocr3
 // RMNSignatures, if RMN is configured for some lanes involved in the commitment.
 // A report with RMN signatures but without merkle roots is invalid.
 type CommitPluginReport struct {
-	MerkleRoots   []MerkleRootChain   `json:"merkleRoots"`
-	PriceUpdates  PriceUpdates        `json:"priceUpdates"`
+	MerkleRoots  []MerkleRootChain `json:"merkleRoots"`
+	PriceUpdates PriceUpdates      `json:"priceUpdates"`
+	// RMNSignatures are the ECDSA signatures from the RMN signing nodes on the RMNReport structure.
+	// For more details see the contract here: https://github.com/smartcontractkit/chainlink/blob/7ba0f37134a618375542079ff1805fe2224d7916/contracts/src/v0.8/ccip/interfaces/IRMNV2.sol#L8-L12
 	RMNSignatures []RMNECDSASignature `json:"rmnSignatures"`
 }
 
-// Deprecated: don't use this constructor, just create a CommitPluginReport struct directly.
-// Will be removed in a future version once all uses have been replaced.
-func NewCommitPluginReport(merkleRoots []MerkleRootChain, tokenPriceUpdates []TokenPrice, gasPriceUpdate []GasPriceChain) CommitPluginReport {
```

Comment:
Note to other reviewers: no longer needed and not a breaking change (all call occurrences have been removed already)